rom/dev/src/start.S

```diff
@@ -77,6 +77,17 @@ _start:
     li x1, 0xAAAAA0A9
     csrw 0x7C0, x1
 
+#ifdef CARGO_FEATURE_FPGA_REALTIME
```

Comment:
Is this a GitHub issue for this? Maybe reference it here?

Reply:
Added new GitHub issue to track ROM/ICCM execution issues. Bharat the previously filed one.